frontend/src/components/registerPrompt/index.js

```diff
@@ -48,14 +48,7 @@ class RegisterPrompt extends Component {
         this.goToProfile(res.data.user.profile.display_name)
       })
       .catch((res) => {
-        const { username, email, password, captcha } = res.response?.data?.errors
-
-        this.setState({
-          username: username || [],
-          email: email || [],
-          password: password || [],
-          generalError: captcha || [],
-        })
+        this.handleError(res?.response?.data?.message)
```

Comment:
Isn't res always a response?

```suggestion
        this.handleError(res.response?.data?.message)
```